contracts/Core/AssetManager.sol

```diff
@@ -56,8 +66,6 @@ contract AssetManager is ACL, AssetStorage {
         string name,
         uint32 aggregationMethod,
         uint256[] jobIDs,
-        address creator,
-        uint256 credit,
```

Comment:
Earlier in job we had credit as param, which I suppsose was for reward for that job, but I didnt find it being transfered anywhere, Was it being used ?

Reply:
That functionality isnt being used anywhere and credit was there as fee a user has to pay to create a job but since this can only be called by the admin only, there is no point in having credit and creator.

Reply:
yep